eng/versioning/version_data.txt

```diff
@@ -23,21 +23,21 @@ com.microsoft.azure.cognitiveservices:azure-cognitiveservices-contentmoderator;1
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-customvision-prediction;1.1.0-beta.3;1.1.0-beta.3
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-customvision-training;1.1.0-beta.3;1.1.0-beta.3
 com.microsoft.azure.cognitiveservices:azure-cognitiveservices-faceapi;1.1.0-beta.1;1.1.0-beta.1
-com.microsoft.azure:azure-cosmos-parent;3.3.1;3.4.0
-com.microsoft.azure:azure-cosmos;3.3.1;3.4.0
-com.microsoft.azure:azure-cosmos-benchmark;3.3.1;3.4.0
+com.microsoft.azure:azure-cosmos-parent;3.4.0;3.5.0
+com.microsoft.azure:azure-cosmos;3.4.0;3.5.0
+com.microsoft.azure:azure-cosmos-benchmark;3.4.0;3.5.0
 com.microsoft.azure:azure-cosmos-examples;3.3.1;3.4.0
```

Comment:
Why is this left untouched ?
Shouldn't azure-cosmos-examples be changed to 3.4.0 and 3.5.0?

Reply:
@kushagraThapar com.microsoft.azure:azure-cosmos-examples only has a 3.3.1 release, 3.4.0 has not been released so this version did not need to be changed.

Reply:
As per our conversation I'll change the versions of benchmark/examples to the following with the next commit to this PR.
com.microsoft.azure:azure-cosmos-benchmark;3.4.0;3.4.0
com.microsoft.azure:azure-cosmos-examples;3.4.0;3.4.0